actiontext/lib/action_text/engine.rb

```diff
@@ -55,6 +55,14 @@ def to_trix_content_attachment_partial_path
       end
     end
 
+    initializer "action_text.fixture_set_helper" do
+      ActiveSupport.on_load(:active_support_test_case) do
+        require "active_record/fixtures"
+
+        ActiveRecord::FixtureSet.context_class.include ActionText::FixtureFileHelpers
```

Comment:
This will likely need some guidance.

I tried to call `FixtureSet.context_class...` without the `ActiveRecord` scope, but had autoloading issues. Is there a better way?